.github/workflows/cd.yml

```
@@ -8,14 +8,20 @@ on:
     workflows: ["Continuous Integration"]
     types: [completed]
+    branches: [main]
 
 jobs:
   deploy:
     runs-on: ubuntu-latest
     permissions:
       actions: read
       contents: write
-    if: ${{ github.event.workflow_run.conclusion == 'success' && github.event.workflow_run.event == 'push' }}
+    if: >
```

actionlint (1.7.12)

[error] if: condition "${{\n  github.event.workflow_run.conclusion == 'success' &&\n  github.event.workflow_run.event == 'push' &&\n  github.event.workflow_run.head_branch == 'main'\n}}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)